GerritDownloader/src/main/java/com/holmsted/gerrit/CommandLineParser.java

```diff
@@ -110,6 +115,14 @@ public String getServerName() {
     }
 
     @Nullable
+    public String getPrivateKey() {
+        if (privateKey != null) {
+          return privateKey;
+        }
+        return "~/.ssh/id_rsa";
```

Comment:
Yes and it worked for me.  I'm guessing it's because the shell calling ssh is performing the expansing.  Regardless, I've updated the code to match what you done in resolveOutputDir() so if shell expansion is not supported it should still work as expected.